english/android/marshmallow6.0.md

```diff
@@ -8,4 +8,5 @@ Check the updates for your smartphone now!
 There are two options here:
 
 1. You have an update, and won't read this further;
-2. You are unlucky one, and you better [choose another path](../marshmallow.md);
\ No newline at end of file
+2. You are unlucky one, and you better [choose another path](../marshmallow.md);
+3. to view new android jetPack [click here](https://www.youtube.com/watch?v=IrMw7MEgADk&t=1562s)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.